images/runtime/dotnetcore/Dockerfile.template

```diff
@@ -16,7 +16,8 @@ FROM %DOTNETCORE_BASE_IMAGE%
 # a version of `curl` that has known issues.
 # We manually update it here so we can still depend on the original images.
 # This command should be removed once support for deprecated .NET core images is halted.
-RUN apt update && apt install curl --yes
+RUN sed -i '/jessie-updates/d' /etc/apt/sources.list  # Now archived
```

Comment:
I got this workaround from here: https://github.com/debuerreotype/docker-debian-artifacts/issues/66#issuecomment-476616579 which seems to have fixed the issue

Reply:
Just FYI...the original error was:

```
Step 9/16 : FROM microsoft/dotnet:1.0.14-runtime

 ---> fdca09b2904f

Step 10/16 : RUN apt update && apt install curl --yes

 ---> Running in 055c0cf2b55b


 


WARNING: apt does not have a stable CLI interface yet. Use with caution in scripts.


 


Ign http:&#x2F;&#x2F;deb.debian.org jessie InRelease‌

Ign http://deb.debian.org jessie-updates InRelease

Get:1 http://deb.debian.org jessie Release.gpg [2420 B]

Ign http://deb.debian.org jessie-updates Release.gpg

Get:2 http://deb.debian.org jessie Release [148 kB]

Get:3 http://security.debian.org jessie/updates InRelease [44.9 kB]

Ign http://deb.debian.org jessie-updates Release

Get:4 http://deb.debian.org jessie/main amd64 Packages [9098 kB]

Err http://deb.debian.org jessie-updates/main amd64 Packages

  

Get:5 http://security.debian.org jessie/updates/main amd64 Packages [822 kB]

Err http://deb.debian.org jessie-updates/main amd64 Packages

  

Err http://deb.debian.org jessie-updates/main amd64 Packages

  

Err http://deb.debian.org jessie-updates/main amd64 Packages

  

Err http://deb.debian.org jessie-updates/main amd64 Packages

  404  Not Found

Fetched 10.1 MB in 9s (1097 kB/s)

W: Failed to fetch http:&#x2F;&#x2F;deb.debian.org&#x2F;debian&#x2F;dists&#x2F;jessie-updates&#x2F;main&#x2F;binary-amd64&#x2F;Packages  404  Not Found‌


 


E: Some index files failed to download. They have been ignored, or old ones used instead.

The command &#x27;&#x2F;bin&#x2F;sh -c apt update &amp;&amp; apt install curl --yes&#x27; returned a non-zero code: 100‌
```